test/no-hot/no-hot.test.js

```diff
@@ -1,10 +1,17 @@
 'use strict';
 const { run } = require('../utils/test-utils');
 const { stat, readFile } = require('fs');
-const { resolve } = require('path');
+const { resolve, join } = require('path');
+const rimraf = require('rimraf');
 const { yellow } = require('colorette');
 
 describe('no-hot flag', () => {
+    beforeEach((done) => {
+        rimraf(join(__dirname, './bin/*'), () => {
+            done();
+        });
+    });
```

Comment:
Yep, we need move it for global options and avoid using in each test, maybe https://jestjs.io/docs/en/configuration#globalteardown-string?

Reply:
`globalTearDown` runs after each `test suite` we need to run it before/after each `test case`.

Reply:
We already clean after each `test suite`. 

https://github.com/webpack/webpack-cli/blob/0038ff282931748903fe746d1ce79b5dbf333a80/jest.config.js#L19

Reply:
cab you try with `global.beforeEach`?

Reply:
Can you point me to the corresponding docs of `global.beforeEach`?

Reply:
Just try defining before each in https://github.com/webpack/webpack-cli/blob/master/setupTest.js

Reply:
![Screenshot at 2020-11-04 18-20-24](https://user-images.githubusercontent.com/46647141/98113953-7203a280-1eca-11eb-933c-ea55a91819a5.png)

Reply:
try `globals.beforeEach`?

Reply:
or maybe just `beforeEach`

Reply:
https://github.com/facebook/jest/blob/master/docs/Configuration.md#injectglobals-boolean

Reply:
Working with global 😄 .

```
const { beforeEach } = require('@jest/globals')
const rimraf = require('rimraf');
const { join } = require('path');
jest.setTimeout(240000);

beforeEach((done) => {
    rimraf(join(__dirname, './test/mode/mode-with-config/bin/*'), () => {
        done();
    });
});
```

Reply:
nice nice, let's maintain a list of dirs which are to be deleted before every test